src/utils/constants.ts

```diff
@@ -400,9 +394,12 @@ export function isSupportedChainId(chainId: number): chainId is ChainId {
 export function getConfigStoreAddress(
   chainId: ChainId = hubPoolChainId
 ): string {
-  const configStoreAddress = configStoreAddresses[chainId];
+  const configStoreAddress = utils.getDeployedAddress(
+    "AcrossConfigStore",
+    chainId
+  );
   assert(
-    configStoreAddress !== AddressZero,
+    Boolean(configStoreAddress),
```

Comment:
Is this asserting that the `configStoreAddress` exists? if so, we may want to use `sdk.utils.isDefined(configStoreAddress)`
```suggestion
    sdk.utils.isDefined(configStoreAddress),
```

Reply:
Yea, this should assert that the `configStoreAddress` is truthy. Hmm, what exactly does `isDefined` check though? Why is using that helper better than just casting to a boolean?

Reply:
Actually, I think checking whether it is a valid address makes the most sense. Changed here https://github.com/across-protocol/frontend-v2/pull/815/commits/38f86af85ce747f29cfee58fdab366f7d5a117d9